cms/static/js_test.yml

```diff
@@ -56,7 +56,7 @@ lib_paths:
     - xmodule_js/common_static/js/vendor/jQuery-File-Upload/js
     - xmodule_js/src/xmodule.js
     - xmodule_js/common_static/js/test/i18n.js
-    - xmodule_js/common_static/js/vendor/draggabilly.pkgd.js
+    - xmodule_js/common_static/js/vendor/draggabilly.js
```

Comment:
I don't understand this change because the name of the file in common/static/js/vendor is draggabilly.pkgd.js.